setup.py

```diff
@@ -1,17 +1,47 @@
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
+# -*- coding: utf-8 -*-
```

Comment:
It tells python the file is utf-8 encoded, per PEP0263. I typically add it to my stuff, though apparently the more 'standard' format is `# coding=utf-8`. If we add it here it should be in all files. I can punt that to a later pr.